src/extension.ts

```diff
@@ -193,32 +193,35 @@ export async function activate(context: vscode.ExtensionContext): Promise<AzureE
                     // ignore
                 });
             // prompt user to deploy to newly created web app
-            if (await vscode.window.showInformationMessage('Deploy to web app?', yesButton, noButton) === yesButton) {
-                this.properties[deployingToWebApp] = 'true';
-                await deploy(this, false, createdApp);
-            } else {
-                this.properties[deployingToWebApp] = 'false';
-            }
+            vscode.window.showInformationMessage('Deploy to web app?', yesButton, noButton).then(
+                async (input: vscode.MessageItem) => {
+                    if (input === yesButton) {
+                        this.properties[deployingToWebApp] = 'true';
```

Comment:
I'm fine with not awaiting this (that's why `showInformationMessage` isn't on the shared AzureUserInput since information messages should almost never be awaited), but be aware you probably wont' get this telemetry anymore

Reply:
I'm not sure if this very useful telemetry actually-- I think I'm okay with losing it.

Reply:
If you're okay with losing it, I would prefer to remove the telemetry rather than get inconsistent telemetry